autopush/db.py

```diff
@@ -96,15 +98,30 @@
 TRACK_DB_CALLS = False
 DB_CALLS = []
 
-# See https://botocore.readthedocs.io/en/stable/reference/config.html for
-# additional config options
-g_dynamodb = boto3.resource(
-    'dynamodb',
-    config=botocore.config.Config(
-        region_name=os.getenv("AWS_REGION_NAME", "us-east-1")
-    )
-)
-g_client = g_dynamodb.meta.client
+SESSION_ARGS = {}
+MAX_SESSIONS = 10
+
+
+class Sessions(object):
+
+    def __init__(self, conf=SESSION_ARGS):
+        self.pool = deque(maxlen=MAX_SESSIONS)
+        session = boto3.session.Session()
+        if not conf.get("endpoint_url") and os.getenv("AWS_LOCAL_DYNAMODB"):
```

Comment:
```
"endpoint_url" not in conf
```

Reply:
It normally shouldn't be. If it's not specified, boto3 will use it's own, else it will use whatever you note. (Passing `None` is problematic) We do pass an `endpoint_url` as part of testing.